src/components/Order/OrderAction/orderAction.css

```diff
@@ -0,0 +1,80 @@
+.ask,
```

Comment:
do you really need css? the styled-components approach is preferable

there are some caveats described here https://www.styled-components.com/docs/api#caveat-with-classname but it should cover this case.
The thing is that you need to define `className?: string` property and bind className attr of your parent div to the `className` prop, i mean `<div className={this.props.className}>`